.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,71 @@
+name: Update gem RBIs on Dependabot PRs
+
+on:
+  workflow_call:
+    inputs:
+      dependency_setup_command:
+        description: A bash script that sets up manually-required dependencies
+        type: string
+        required: false
+      tapioca_env_vars:
+        description: A space-delimited list of environment variables and their values, to be passed to the `tapioca gem` command
+        type: string
+        required: false
+      valid_actors:
+        description: A stringified list of GitHub actors that can trigger this job
+        type: string
+        default: '["dependabot[bot]"]'
+        required: false
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-gem-rbis:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    #if: ${{ contains(fromJson(inputs.valid_actors), github.actor) }}
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Set up any manually specified dependencies
+        if: ${{ inputs.dependency_setup_command != '' }}
+        run: ${{ inputs.dependency_setup_command }}
+        env:
+          DEBIAN_FRONTEND: noninteractive
+      - uses: ruby/setup-ruby@v1
+        id: setup
+        with:
+          bundler-cache: true
+          ruby-version: 3.2
+      - name: Ruby setup error details
+        if: ${{ failure() && steps.setup.conclusion == 'failure' }}
+        run: |
+          echo "A failure during the Ruby setup step could indicate that the permissions" \
+            " for this repo are not configured correctly." \
+          exit 1
+      - name: Update gem RBIs
+        run: |
```

Comment:
Should we also run `tapoica dsl` here?

Reply:
I don't think it's necessary. We haven't been using DSL RBIs in the Ruby LSP.

Reply:
I was thinking more for if we generalized this to be a shared action.

Reply:
Wouldn't that fail for Ruby-LSP?

irrc when the project is not a Rails app `tapioca dsl --verify` will return the exit code 1 with this error:

```
No classes/modules can be matched for RBI generation.
Please check that the requested classes/modules include processable DSL methods.
```

Reply:
Ah yes it fails, I thought it would just be a no-op if there weren't any.

Isn't is possible to have DSL compilers in non-Rails projects?

Reply:
Not at the moment but we're working on it.

If you want to make this sharable, you _could_ look for the `config/application.rb` file to exist before running `tapicoa dsl`.

Reply:
I really don't think running `tapioca dsl` is necessary. And for non simple apps this will fail because we can't boot it. IMO if someone wants to update DSLs they can copy the workflow and modify it. We want a workflow that's easy to maintain for us and just works.

Reply:
Fair enough, removed to keep things simple for now.